1.0.0 doc changes

cs-abhishek-shukla opened the pull request:
- Navigation buttons at the bottom of each .md file
- Fixed Typo, Grammar, and information flow issues

cs-rohit-khune reviewed (approved):
LGTM

cs-abhinav-goel commented on docs/contents.md:
```diff
@@ -5,33 +5,39 @@
+|:------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Investigate Brute Force Attempt                 | Investigates login failures and also identifies other impacted assets that have been victims of the brute force attempts from a particular source of attack                |
+| Investigate Brute Force Attempt (FortiSIEM)     | Investigates login failures from FortiSIEM and also identifies other impacted assets that have been victims of the brute force attempts from a particular source of attack |
+| Scenario - FortiSIEM (Brute Force Attack) Alert | Generates a demo alert for Brute Force Attack                                                                                                                              |
```
In this we should add the tag "Renamed"

cs-abhishek-shukla replied:
Redundant. As we are mentioning the same in release notes.

cs-abhinav-goel commented on docs/setup.md:
```diff
@@ -10,10 +10,10 @@
+| Solution Pack Name | Version | Purpose                                                  |
+|:-------------------|:-------:|:---------------------------------------------------------|
+| SOAR Framework     | v1.0.0  | Required for Incident Response modules                   |
+| SOC Simulator      | v1.0.1  | Required for Scenario Module and SOC Simulator connector |
```
SOC simulator version >> 1.0.2
SFSP >> 2.1.0
@cs-rohit-khune @cs-abhishek-shukla : Please confirm.

cs-rohit-khune replied:
The SP will work fine on lower versions of SFSP and SOC SP.  We haven't made any version-dependent changes.

cs-suyog-jinturkar commented on docs/setup.md:
```diff
@@ -10,10 +10,10 @@
-|SOC Simulator 1.0.1|Required for Scenario Module and SOC Simulator connector|
+| Solution Pack Name | Version | Purpose                                                  |
+|:-------------------|:-------:|:---------------------------------------------------------|
+| SOAR Framework     | v1.0.0  | Required for Incident Response modules                   |
```
or later

cs-suyog-jinturkar commented on docs/setup.md:
```diff
@@ -10,10 +10,10 @@
+| Solution Pack Name | Version | Purpose                                                  |
+|:-------------------|:-------:|:---------------------------------------------------------|
+| SOAR Framework     | v1.0.0  | Required for Incident Response modules                   |
+| SOC Simulator      | v1.0.1  | Required for Scenario Module and SOC Simulator connector |
```
1.0.1 or later - @cs-abhishek-shukla

cs-abhishek-shukla committed:
1.2.0 doc changes
- Navigation buttons at the bottom of each .md file
- Fixed Typo, Grammar, and information flow issues
